indico/modules/rb/forms/reservations.py

```diff
@@ -111,8 +111,6 @@ class NewBookingPeriodForm(NewBookingFormBase):
 
 class NewBookingConfirmForm(NewBookingPeriodForm):
     booked_for_user = PrincipalField(_(u'User'), [DataRequired()], allow_external=True)
-    contact_email = StringField(_(u'Email'), [InputRequired(), IndicoEmail(multi=True)])
-    contact_phone = StringField(_(u'Telephone'))
```

Comment:
We should decide whether to remove contact_email/phone everywhere! For contact_phone it makes sense for sure, but contact_email is actually used to control modification access on the booking...

    def is_booked_for(self, user):
        if user is None:
            return False
        return self.booked_for_user == user or bool(self.contact_emails & set(user.all_emails))

cc @pferreir